### PR TITLE
[REV-24] Create contest detail view

### PR DESCRIPTION
**Pull request by g-esteves-rr**

### CODE REVIEW

**Comment by tiagocardoso98** [comment]
The poetry-lock file is lacking django-filter dependencies. In order to run the backend in this branch I had to use the command ```poetry add django-filters``` instead of just ```poetry install```.

**Comment by tiagocardoso98** [comment]
Some tests are failing due to the same problem described in [issue 34](https://linear.app/revent/issue/REV-34/factories-update).

**Comment by g-esteves-rr** [comment]
> 

I've Updated the poetry.lock with the django-filters

**Review by tiagocardoso98 — APPROVED** [review]
I believe that the testing issue should be resolved on a separate PR and that this PR is ready for merge.

File: photo/views.py [hunk 1]
```diff
@@ -18,24 +21,30 @@ class UserViewSet(viewsets.ModelViewSet):
...
-@api_view(["GET"])
-def submissions_from_contest(request, id):
-    submissions = Submission.objects.filter(contest=id)
-    serializer = SubmissionSerializer(submissions, many=True)
-    return Response(serializer.data)
+
+# @api_view(["GET"])
+# def submissions_from_contest(request, id):
+#     contest = Contest.objects.filter(pk=id)
+#     if(len(contest) == 0):
```
Comment (andrevieira11) [inline photo/views.py:46]: we can probably remove this?

**Review by andrevieira11 — APPROVED** [review]

**Review by kenvontucky — APPROVED** [review]

**Pull request merged**